### ISSUE TITLE
Suggest missing tags/properties that could aid accessibility

### ISSUE THREAD

**Issue by mrlacey** [mrlacey/Rapid-XAML-Toolkit#137]
What can be done to help developers produce more accessible XAML?

Ideas include missing AutomationProperty tags. Need to establish, rules for adding these though and then look at what else could be added too.

**Comment by mrlacey** [comment]
Need to consider how to add these in ways that support localization too.

From elsewhere:

> "Support for the Windows Narrator is quite a job for adding ControlName.[using:Windows.UI.Xaml.Automation]AutomationProperties for each meaningful controls. Adding a tooltip is also highly suggested. The syntax is long and it can be error prone. I would love to see something more user friendly for the dev."

**Comment by mrlacey** [comment]
X-Ref: https://github.com/microsoft/axe-windows/blob/7e01ebfc1d74b8c72ebcc7894b43988a5421023f/docs/RulesDescription.md

**Comment by mrlacey** [comment]
X-Ref #242
